Packs/Core/TestPlaybooks/playbook-T1036_-_Masquerading_-_Test.yml

```diff
@@ -0,0 +1,589 @@
+  - Validate the actions that were taken exist in the incident context.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 8bd038f8-98ff-49f1-8dd8-3b6d1868c638
+    type: start
+    task:
+      id: 8bd038f8-98ff-49f1-8dd8-3b6d1868c638
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "12"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 80
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: b2edc4a5-9adc-465b-8fda-85f054ce148e
+    type: playbook
+    task:
+      id: b2edc4a5-9adc-465b-8fda-85f054ce148e
+      version: -1
+      name: T1036 - Masquerading
+      description: |-
+        This playbook handles masquerading alerts based on the MITRE T1036 technique.
+        An attacker might leverage Microsoft Windows well-known image names to run malicious processes without being caught.
+
+        **Attacker's Goals:**
+
+        An attacker is attempting to masquerade as standard windows images by using a trusted name to execute malicious code.
+
+        **Investigative Actions:**
+
+        Investigate the executed process image and verify if it is malicious using:
+
+        * XDR trusted signers
+        * VT trusted signers
+        * VT detection rate
+        * NSRL DB
+
+        **Response Actions**
+
+        The playbook's first response action is a containment plan which is based on the initial data provided within the alert. In that phase, the playbook will execute:
+
+        * Auto block indicators
+        * Auto file quarantine
+        * Manual endpoint isolation
+
+        When the playbook executes, it checks for additional activity using the Endpoint Investigation Plan playbook, and another phase, which includes containment and eradication, is executed.
+
+        This phase will execute the following containment actions:
+
+        * Manual block indicators
+        * Manual file quarantine
+        * Auto endpoint isolation
+
+        And the following eradication actions:
+
+        * Manual process termination
+        * Manual file deletion
+        * Manual reset of the user’s password
+
+        External resources:
+
+        [MITRE Technique T1036](https://attack.mitre.org/techniques/T1036/)
+
+        [Possible Microsoft process masquerading](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR-Analytics-Alert-Reference/Possible-Microsoft-process-masquerading)
+      playbookName: T1036 - Masquerading
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "24"
+    scriptarguments:
+      AutoCloseAlert:
+        simple: "True"
+      AutoContainment:
```

Comment:
if the hash doesn't exist, why do we need to test the containment here? you have a dedicated PB for this .... or I'm missing a point here

Reply:
The playbook checks if the hash was added to the block list.